scipy/stats/_sensitivity_analysis.py

```diff
@@ -223,23 +229,29 @@ def sobol_indices(
     Parameters
     ----------
     func : callable or dict(str, array_like)
-        Function to compute the Sobol' indices from. It's signature must be
-        ``func(x: ArrayLike) -> ArrayLike``, with ``x`` of shape ``(n, d)``
-        and the output should have a shape ``(n, s)`` with ``s`` the number
-        of output and ``n`` the number of samples.
+        Function to compute the Sobol' indices from. It's signature must be::
+
+            func(x: ArrayLike) -> ArrayLike
+
+        with ``x`` of shape ``(n, d)`` and the output should have a shape
+        ``(n, s)`` with ``s`` the number of outputs and ``n`` the number of
+        samples.
     n : int
-        Must be a power of 2. The total number of function call will be
+        Number of samples. 
+        Must be a power of 2. The total number of function calls will be
         ``n(d+2)``.
     dists : list(distributions), optional
-        Must be specified if `func` is a callable, and has no use otherwise.
-
         List of each parameter's marginal distribution. Each parameter being
         independently distributed.
 
-        Distributions must be an instance of a class with a ``pdf`` or
+        Distributions must provide a numerical inverse. They are either
```

Comment:
? This doesn't seem to agree with the next sentence, unless ``pdf`` is a "numerical inverse".
```suggestion
        They are either
```